content/en/dashboards/widgets/wildcard.md

```diff
@@ -0,0 +1,144 @@
+
+ ## Setup
+
+ ### Configure a new Wildcard widget
+1. [Check native widgets][4]. See if a Datadog widget can fulfill your requirements.
+1. If no Datadog widget meets your requirements, in a new or pre-existing dashboard, click **Add Widgets**.
+1. Click and drag the Wildcard Widget icon from the widget tray. 
+1. Copy a Vega-Lite Definition from the [public gallery][5] to find a starter Vega-Lite specification. 
+1. Open the Wildcard widget [full screen editor][6] and click **Define Visual**. 1. Paste the copied Vega-Lite definition.
+1. Click **Apply** to apply your configuration changes, see a preview of the visualization, and iterate on your design. **Note**: You must click **Apply** to add your changes, however this does not save your configuration.
+1. (Optional) Debug Vega-Lite specification mismatches with [Data Preview](#data-preview). Make sure the query in your Vega-Lite specification maps to the Datadog query. 
+1. Click **Save**.
+
+### Import data from an existing widget
+
+1. Copy from an existing Datadog widget using `cmd+c`.
+1. Open the Wildcard widget [full screen editor][6].
+1. Paste with `cmd+v`.
+1. Click Save.
+
+## Command palette
+
+{{< img src="/dashboards/widgets/wildcard/command_palette.png" alt="Command palette modal showing the ability to search commands and autoselect chart" style="width:100%;" >}}
+
+The command palette provides quick access to Wildcard widget tools. Activate with `cmd + shift + p` or click the info icon at the top of the page. 
+
+## Data Preview
+
+{{< img src="/dashboards/widgets/wildcard/data_preview_arrow_icon.png" alt="Highlighting the arrow icon to access the Data Preview panel" style="width:100%;" >}}
+
+The Data Preview table shows the response, fields, and values from your data request that are available to use in your Vega-lite specification. To access, click the arrow at the bottom of the Wildcard widget editor to *Show data preview*. There are three types of tables in the preview:
+- Request Rows: Displays your actual data.
+- Request Columns: Displays column summary statistics and data types. 
+- Internal Tables: Displays transformed data stored by Vega-Lite.
+
+Datadog widgets automatically map the query results to the visualization elements, but the Wildcard widget requires you to add a custom Vega-Lite specification that defines how the Datadog query maps to visual elements. This creates the potential for a mismatch. With Data Preview, you can verify that the Vega-Lite specification maps to the correct query response.
+
+To highlight this, use the example metric query of `system.cpu.user` averaged by `env`
+
+{{< img src="/dashboards/widgets/wildcard/example_configuration_query.png" alt="Example widget configuration metric query for system.cpu.user grouped by env" style="width:100%;" >}}
+
+Notice the matching **query1** and **env** fields listed in the Vega-Lite specification and the Data Preview column. 
```

Comment:
It wasn't immediately clear to me what this JSON represents - and the highlighted values are `env` and `example`, so I wasn't sure where `query1` came in. I think you could probably add a little more signposting around these examples to explain what you're showing the user. Is the point that the visualization starts out broken and then you fix it by reassigning the alias? If this is the case, it's useful to let the user know you're going to show them a broken example before the example appears.

Sidenote: I like the use of tables here.

Reply:
Copy paste error 😞 This should be `query1`.  I agree this could be broken down more. I added a new section specifically to map values between Datadog and Vega-Lite, hopefully that makes it clearer!